app/src/main/java/pl/animagia/file/FileUrl.java

```diff
@@ -0,0 +1,42 @@
+package pl.animagia.file;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.StringReader;
+
+public class FileUrl {
+
```

Comment:
Add a private empty constructor since there's no need to create instances of this class